packages/manager/apps/carrier-sip/src/routing.js

```diff
@@ -12,6 +12,13 @@ export default /* @ngInject */ ($stateProvider) => {
       ) => CarrierSipService.getCarrierSip(billingAccount, serviceName),
       cdrsLink: /* @ngInject */ ($state, billingAccount, serviceName) =>
         $state.href('app.cdrs', { billingAccount, serviceName }),
+      clusterDetails: /* @ngInject */ ($http, billingAccount, serviceName) =>
+        $http
+          .get(
+            `/telephony/${billingAccount}/carrierSip/${serviceName}/clusterDetails`,
+          )
+          .then(({ data }) => data)
+          .catch(() => ({})),
```

Comment:
can you differentiate some error types such as 404 or 500 ? or does the api returns some weird error code ?

Reply:
done :)